modAionImpl/src/org/aion/zero/impl/AionHub.java

```diff
@@ -122,21 +125,35 @@ public static AionHub inst() {
     private static final int INIT_ERROR_EXIT_CODE = -1;
 
     public AionHub() {
+        initializeHub(CfgAion.inst(), AionBlockchainImpl.inst(), AionRepositoryImpl.inst(), false);
+    }
+
+    private void initializeHub(
```

Comment:
Could the method body of initalizeHub can't just directly live in the constructor `AionHub(CfgAion, AionBlockchainImpl, AionRepositoryImpl, boolean)` ?  Then the zero-arg constructor can call that constructor